include/types.h

```diff
@@ -28,10 +28,14 @@ SOFTWARE.
 #define TYPES_HPP
 
 // precision of AMG storage
-#if 0
+#if 1
 #define pfloat float
+#define pfloatString "float"
+#define pfloatSize 4
```

Comment:
I wouldnt think we need a define for the type size. If this is used in some `#if` pre-processor checks, we can probably have another way to check